utils.js

```diff
@@ -60,7 +60,11 @@ export const dataToResponse = (
 	};
 };
 
-export const dataToEmbeddingResponse = (output) => {
+export const dataToEmbeddingResponse = (
```

Comment:
same comment h ere about completion token, total_tokens would just be prompt_tokens i guess

Reply:
Removed and updated.